### ISSUE TITLE
play options missing when long pressing "folder tree mode" in music!

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kader990/android-xbmcremote#140]
```
What steps will reproduce the problem?
1. go to listen to music
2. select the last sorting option (folder view)
3. long pressing any folder has no functionality!

What is the expected output? What do you see instead?

I hoped it would give me a "Play selected, Enqueue selected, etc etc" menu
like in the "Artist" and other views but nothing!  You can only play single
files.  I'd really like to be able to click a folder or a file in the
folder and have it queue the entire folder.

What version of XBMC Remote are you using?

xbmc 9.11 r26017

Which Android device are you using (and which ROM, if custom)?

Droid 

Which XBMC revision are you running running and on on which platform?

WinXP Camelot


Please provide any additional information below.

To summarize, I'd really appreciate submenu functionality when long
pressing any folder in the folder tree (as all my albums are already sorted
on my hard drive).  On a side note, an option to disable the remote
changing orientation when the phone is sideways would be excellent as well.
 The file lists malfunction when the phone is rotated.

EXCELLENT application though...this is my greatest find of 2010 so far. 
Keep up the great work...making my life easier.
```

Original issue reported on code.google.com by `mpinte...@gmail.com` on 1 Jan 2010 at 9:38
- Merged into: #114
